rotkehlchen/chain/evm/decoding/stakedao/decoder.py

```diff
@@ -125,14 +126,28 @@ def _decode_claim(
 
         return DEFAULT_DECODING_OUTPUT
 
-    def _decode_claim_with_bounty(self, context: DecoderContext) -> DecodingOutput:
-        if context.tx_log.topics[0] != CLAIMED_WITH_BOUNTY:
-            return DEFAULT_DECODING_OUTPUT
+    def _decode_reward_claim_events(self, context: DecoderContext) -> DecodingOutput:
+        if context.tx_log.topics[0] == CLAIMED_WITH_BOUNTY:
+            reward_token_address = bytes_to_address(context.tx_log.data[0:32])
+            amount = int.from_bytes(context.tx_log.data[32:64])
+            period = Timestamp(int.from_bytes(context.tx_log.data[96:128]))
+            return self._decode_claim(context=context, reward_token_address=reward_token_address, amount=amount, period=period)  # noqa: E501
+        elif context.tx_log.topics[0] == REWARDS_CLAIMED_TOPIC:
+            gauge_addresses = {
+                bytes_to_address(context.tx_log.data[x:x + 32])
+                for x in range(64, len(context.tx_log.data), 32)  # the first 64 bytes are ABI metadata (offset + array length), so we skip them.  # noqa: E501
```

Comment:
so this can have many gauge addresses? What if the length of context.tx_log.data does not add up to the size of gauges?